osm2gtfs.py

```diff
@@ -103,12 +103,7 @@ def main():
     stops = osmhelper.get_stops(routes)
 
     schedule = transitfeed.Schedule()
-    agency = schedule.AddAgency(
-        name=config['agency']['agency_name'],
-        url=config['agency']['agency_url'],
-        timezone=config['agency']['agency_timezone'],
-        agency_id=config['agency']['agency_id']
-    )
+    agency = load_agency(config)
```

Comment:
This will break the script, because the agency is never added to the schedule object. Did you test this?